### PR TITLE
[iOS] Add temp cindex attribute to build on xcode 15

### PR DESCRIPTION
**Pull request by steveisok**
Without CursorKind 437, the cross build for iOS/tvOS and iOS/tvOSSimulator will fail. This change adds a temporary attribute to avoid that.

### CODE REVIEW

File: src/tasks/AppleAppBuilder/AppleAppBuilder.cs [hunk 1]
```diff
@@ -223,6 +223,7 @@ public void ValidateRuntimeSelection()
 
     public override bool Execute()
     {
+        bool shouldStaticLink = true;
```
Comment (vargaz) [inline src/tasks/AppleAppBuilder/AppleAppBuilder.cs:226]: Is this related ?

Comment (steveisok) [reply src/tasks/AppleAppBuilder/AppleAppBuilder.cs:226]: Yeah, some version of xcode changed simulator behavior where if any dylib is deployed with the app it'll try to load it first.  This was causing problems because we always static link the components.

**Review by vargaz — APPROVED** [review]
